.circleci/config.yml

```diff
@@ -9,6 +9,10 @@ job-defaults: &job-defaults
     - AWS_ACCOUNT_ID: "883127560329"
     - AWS_PROFILE: dev/vault
 
+attach-workspace: &attach-workspace
```

Comment:
By the way, this idiom is not the recommended way to use YAML anchors. You're supposed to use it on the first instance of when the construct appears in the document, this is adding additional unused constructs before the document that do not correspond to any understood schema.